src/components/login/SignupBox/SignupBox.css

```diff
@@ -0,0 +1,54 @@
+.auth-box-signup {
+    padding: 1rem 1.5rem 2.5rem 1.5rem;
+    height: auto;
+    width: 30%;
+    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
```

Comment:
here also if you are using same shadow you can create class for that

Reply:
ok noted